src/BulletSoftBody/btDeformableContactConstraint.cpp

```diff
@@ -256,6 +256,8 @@ btScalar btDeformableRigidContactConstraint::solveConstraint()
     impulse = impulse_normal + impulse_tangent;
     // apply impulse to deformable nodes involved and change their velocities
     applyImpulse(impulse);
+	if (residualSquare < 1e-7)
```

Comment:
Could you describe the idea behind this early out?

Reply:
This is actually a bug, not a serious one though. I'll fix this in the next PR.